addons/mail/static/src/core/common/persona_model.js

```diff
@@ -96,7 +96,7 @@ export class Persona extends Record {
     isInternalUser = false;
     /** @type {luxon.DateTime} */
     write_date = Record.attr(undefined, { type: "datetime" });
-    groups_id = Record.many("res.groups", { inverse: "personas" });
+    group_ids = Record.many("res.groups", { inverse: "personas" });
```

Comment:
Here to rename to `all_group_ids`. I don't think we ever use/need the explicitly assigned groups here.